build-system/babel-plugins/babel-plugin-optimize-objstr/test/fixtures/transform-assertions/transform-mixed/output.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"" + (keyIsValueId ? " " + "keyIsValueId" : "");
+"" + (false ? " " + 'quoted' : "");
+"" + (true ? " " + "unquoted" : "");
+"" + (probably ? " " + "identifierExpression" : "");
```

Comment:
Shouldn't this be:
```suggestion
"" + (probably ? " " + identifierExpression : "");
```

Reply:
Good catch! Fixed using your suggestion below.